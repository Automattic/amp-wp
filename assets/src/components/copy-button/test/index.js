/**
 * External dependencies
 */
import { create } from 'react-test-renderer';

/**
 * Internal dependencies
 */
import { CopyButton } from '..';

describe( 'CopyButton', () => {
	it( 'matches snapshots', () => {
		// Normal listing.
		const wrapper = create(
			<CopyButton value="Some Text" />,
		);

		expect( wrapper.toJSON() ).toMatchSnapshot();
	} );
} );
