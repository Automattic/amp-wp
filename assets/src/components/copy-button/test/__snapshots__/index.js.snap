// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyButton matches snapshots 1`] = `
<button
  aria-describedby={null}
  className="components-button components-button--copy"
  onClick={[Function]}
  type="button"
>
  Copy
</button>
`;
